tests/settings/SettingTest.py

```diff
@@ -28,6 +29,11 @@ def test_path(self):
                               origin="test" + os.path.sep),
                          os.path.abspath(os.path.join("test", "22")))
 
+    def test_glob(self):
+        self.uut = Setting("key", ".",
+                           origin="test (1)" + os.path.sep + "somefile")
+        self.assertEqual(glob(self.uut), os.path.abspath("test [(]1[)]"))
```

Comment:
you need to make sure that everything done by the abspath function is escaped too!